crypto/asn1/x_int64.c

```diff
@@ -65,9 +73,13 @@ static int uint64_c2i(ASN1_VALUE **pval, const unsigned char *cont, int len,
                     int utype, char *free_cont, const ASN1_ITEM *it)
 {
     uint64_t utmp = 0;
-    char *cp = (char *)pval;
+    char *cp;
     int neg = 0;
 
+    if (*pval == NULL && !uint64_new(pval, it))
```

Comment:
Actually, yes. `fuzz/asn1.c` exercises these types as non-embedded. That's how I discovered the need for this very line